internal/models/types.go

```
@@ -1,88 +1,51 @@
-	// Time range
-	FirstSeen time.Time `json:"first_seen"`
-	LastSeen  time.Time `json:"last_seen"`
-}
-
-// MetricsResult represents the final output of the analysis
-type MetricsResult struct {
-	Summary Summary      `json:"summary"`
-	Paths   []PathMetric `json:"paths"`
+// PathMetrics represents aggregated metrics for a specific path
+type PathMetrics struct {
+	Path              string                 `json:"path"`
```

golangci-lint (1.64.8)

File is not properly formatted

(gofmt)